superset/jinja_context.py

```diff
@@ -284,6 +285,56 @@ def url_param(
             self.cache_key_wrapper(result)
         return result
 
+    def get_guest_user_attribute(
+        self,
+        attribute_name: str,
+        default: str | None = None,
```

Comment:
I think technically the type could be any of the native json types (bool, number, string or another json node). In your example one of your attribute is a `bool` for instance.